lib/webdriveragent.js

```diff
@@ -25,6 +24,37 @@ const WDA_AGENT_PORT = 8100;
 const WDA_BASE_URL = 'http://localhost';
 const BUILD_TEST_DELAY = 1000;
 
+
+class NoSessionProxy extends JWProxy {
```

Comment:
many thanks. Shall I also imp[ort util module from appium-support there? 

Will it work if I import it like this, assuming I save this file as "no-session-proxy.js" into same folder

```
import { NoSessionProxy } from "./no-session-proxy"
```

?